package.json

```diff
@@ -11,7 +11,7 @@
     "debug": "^2.2.0",
     "js-yaml": "^3.5.3",
     "jsonwebtoken": "^5.5.4",
-    "microgateway-core": "^2.0.15",
+    "microgateway-core": "^2.3.2-beta",
```

Comment:
No. `2.1.0` uses the old style of embedding target request and response objects in the `data` object of the function.